porcupine/menubar.py

```diff
@@ -422,6 +427,7 @@ def change_font_size(how: Literal["bigger", "smaller", "reset"]) -> None:
     get_menu("View").add_command(
         label="Reset Font Size", command=partial(change_font_size, "reset")
     )
+    get_menu("View/Focus").add_command(label="Active file", command=focus_active_tab)
```

Comment:
It would be good to disable this when there are no tabs. This way confused users won't think it does something if they browse the menus with no tabs open. You can use `set_enabled_based_on_tab()` to do this.